yarn-project/cli/src/index.ts

```diff
@@ -485,6 +485,27 @@ export function getProgram(log: LogFn, debugLogger: DebugLogger): Command {
       Object.entries(info).map(([key, value]) => log(`${startCase(key)}: ${value}`));
     });
 
+  program
+    .command('inspect-contract')
+    .description('Shows list of external callable functions for a contract')
```

Comment:
nit picky; isn't "external callable" a bit of a tautology?

Reply:
Hmm you could argue that internal functions are callable but from within the contract. I'm not too attached to it though, happy to change it if needed.